ecp-utils.php

```diff
@@ -25,45 +24,36 @@
 	   );
 */
 class ElectrumBCHUtil extends ElectrumUtil {
-
```

Comment:
Can you add these lines back? They're too many and add too much noise. If you really feel the need to remove all these lines, please do it in another separate commit (not pull request) from the addition of the ElectrumBTCUtil. So it's easier to review.

Reply:
I will try to figure out how to resubmit the changes in this file.

Reply:
Hope that is better for you. The plugin is generating a fatal error upon checkout but it was doing that before I started so I do not think my changes caused it